ncm-accounts/src/main/perl/accounts.pm

```diff
@@ -390,14 +395,55 @@ sub apply_profile_groups
     my ($self, $system, $profile) = @_;
 
     while (my ($group, $cfg) = each(%$profile)) {
-      if (!exists($system->{groups}->{$group})) {
+      my $required_members = $cfg->{requiredMembers};
+      $required_members = [] unless $required_members;
+      my @initial_members;
+      if ( exists($system->{groups}->{$group}) ) {
+        if ($system->{groups}->{$group}->{gid} != $cfg->{gid}) {
+          $self->info("Changing gid of group $group to $cfg->{gid}");
+          $system->{groups}->{$group}->{gid} = $cfg->{gid};
+        }
+        @initial_members = keys(%{$system->{groups}->{$group}->{members}});
+        if ( $cfg->{replaceMembers} ) {
+          $self->verbose("Group $group: replacing existing member list");
```

Comment:
i would actaully add a comment here too like
```
# replaceMembers is achieved by initial reset of the members of the group
```
i'm confused a bit by the `replace`, maybe `forceMembers` is better? it's clear in the pod, but still confusing in the code